Tools/setup.csh

```diff
@@ -103,8 +111,18 @@ if ( -f $LEGTOPTAGGERFILE && -l $LEGTOPTAGGERFILE ) then
     rm $LEGTOPTAGGERFILE
 endif
 
+# -p give no error if directory exists
+mkdir -p $STOPCFGDIR
+# -o for overwrite softlinks if they exist
+${SRC}/SusyAnaTools/Tools/scripts/getStopCfg.sh -o -t supp_cfg_tag -d $STOPCFGDIR
```

Comment:
So all the files will go to STOPCFGDIR? This will need some clear up on the code level, as most of the code assume the root file in current directory. Have you tested this? Will it be safe for Condor jobs etc?